src/contents/shared/ProfileEditView.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { EmailSVG } from '../../components/svg/EmailSVG';
+import { NameInputSVG } from "../../components/svg/NameInputSVG";
+import { InputField } from "../../components/form/InputField";
+import { LocationSVG } from "../../components/svg/LocationSVG";
+import { BusinessCardSVG } from '../../components/svg/BusinessCardSVG';
+import { FileAttachmentSVG } from '../../components/svg/FileAttachmentSVG';
+import { CameraSVG } from '../../components/svg/CameraSVG';
+import PropTypes from 'prop-types';
+
+
+export const ProfileEditView = ({ updateUserHandler, name, photoURL, changeHandler, location, user, type, email, handleChangeUpload, progress, url }) => (
+
+	
+	<div className="section-content with-padding">
+
+		<form onSubmit={updateUserHandler}>
+			<h1>Edit your Profile: {user.displayName}</h1>
+			<InputField
+				svg={
+					<NameInputSVG classes="gb-icon gb-icon-medium gb-icon-white inputIcon" />
```

Comment:
add placeholders to all of your input fields

Reply:
Done!